### PR TITLE
Docs - Add API Refs 

### LINKED ISSUE #101: IOTA SDK - Add Python and Node API refs to wiki
The IOTA SDK docs need the Node and Python API references in the wiki and instructions on where to find the Rust API Ref.

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* Generated and added the Python and Node.js API refs to the wiki
* Added an article on where to find the Rust API ref, as well as how to generate it.
* Added an article stating to use the Node.js API ref if working with Wasm.
* Modified the docusarus config so it only skips files starting with `_` in the `_admonitions` folder

## Links to any relevant issues

fixes https://github.com/iotaledger/guild-devx/issues/101

## Type of change

- Documentation Fix
- Documentation Enhancement

## How the change has been tested

Wiki was built locally

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by Thoralf-M — APPROVED** [review]

**Comment by DaughterOfMars** [comment]
Which files were not auto generated?

**Comment by lucas-tortora** [comment]
> Which files were not auto generated?

* [node.js/package.json](https://github.com/iotaledger/iota-sdk/pull/557/files#diff-b604f500ff9583b095808317cfaa0f6ac4f2ee9d010941ddf7af7ddea2105b92)
* [python/readme.md](https://github.com/iotaledger/iota-sdk/pull/557/files#diff-f1115c682f4f652cf0f86395408188731f3d8819b512a9b0ae9fbd34004b977d)
* [python/pydoc-markdown.yml](https://github.com/iotaledger/iota-sdk/pull/557/files#diff-6821021915f0ea9311b83556cdd148e090759a455a6a4b045272678de098784e)
* [documenation/docusarus.config.js](https://github.com/iotaledger/iota-sdk/pull/557/files#diff-1dffb0eac602a458120a09911b873d73fe29d4fc6573349b4693b4cb45f14ac7)
* [documenation/sidebars.js](https://github.com/iotaledger/iota-sdk/pull/557/files#diff-1ec6e68a73832256e38015de77a375739819a73120a855b73095185455b215ee)

Basically anything outside documenation/sdk/docs/references

**Review by thibault-martinez — COMMENTED** [review]
Do we really need to push all that stuff? Can't it be generated by whatever is deploying the docs?

**Comment by lucas-tortora** [comment]
Currently, function params in the Python are listed in plain text, making them very hard to read (see image in thread). 

Would it be alright if I modified the comments to make this an unordered list? I just need to add a `-` or `*` before each param, and some punctuation while am at it. 

Or, better still, does anyone know how to configure pydoc to do this?

image

image

**Comment by lucas-tortora** [comment]
> Do we really need to push all that stuff? Can't it be generated by whatever is deploying the docs?

I don't know how that would work, as this needs to be committed to the repo before publishing, but we can certainly look into it in the future. At the moment, since time is a factor, I replicated the setup we are currently using for wallet.rs, iota.rs, and iota.js. 

Alternatively, if the APIs get published with the packages as in Rust we can just add a link.

**Review by thibault-martinez — APPROVED** [review]
